configure

```diff
@@ -992,6 +994,15 @@ do
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
```

Comment:
why do we have these changes in this PR? Are they related to readme? If not, maybe opening a separate PR for those could be nicer.

Reply:
These keep happening because we people have different versions of autoreconf.

Reply:
I will revert this change

Reply:
reverted